tools/download.sh

```diff
@@ -20,52 +19,52 @@ case "$OSTYPE" in
         # Install curl and wget to download package
         case "$ID" in
             centos*)
-                if [[ -z $(command -v curl) ]]; then
+                if ! hash curl 2>/dev/null; then
                     echo "curl not found, installing..."
                     sudo yum install -y curl
                 fi
-                version=rpm
+
+                package=powershell-6.0.0_alpha.9-1.el7.centos.x86_64.rpm
                 ;;
             ubuntu)
+                if ! hash curl 2>/dev/null; then
+                    echo "curl not found, installing..."
+                    sudo apt-get install -y curl
+                fi
+
                 case "$VERSION_ID" in
                     14.04)
-                        version=ubuntu1.14.04.1_amd64.deb
+                        package=powershell_6.0.0-alpha.9-1ubuntu1.14.04.1_amd64.deb
                         ;;
                     16.04)
-                        version=ubuntu1.16.04.1_amd64.deb
+                        package=powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb
```

Comment:
Line 11:
> release=v6.0.0-alpha.9

Line 37:
> package=powershell_6.0.0-alpha.9-1ubuntu1.14.04.1_amd64.deb

Line 40:
> package=powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb

If we take `6.0.0-alpha.9` part out as a variable, we can update 1 place instead of 3 when needed.